wandb/cli/cli.py

```diff
@@ -1031,6 +893,163 @@ def _parse_settings(settings):
         tuner.run(verbose=verbose)
 
 
+@cli.command(
+    context_settings=CONTEXT,
+    no_args_is_help=True,
+    help="Run a W&B launch sweep (Experimental).",
+)
+@click.pass_context
+@click.option(
+    "--queue",
+    "-q",
+    default=None,
+    help="The name of a queue to push the sweep to",
+)
+@click.option(
+    "--project",
+    "-p",
+    default=None,
+    help="Name of the project which the agent will watch. "
+    "If passed in, will override the project value passed in using a config file",
+)
+@click.option(
+    "--entity",
+    "-e",
+    default=None,
+    help="The entity to use. Defaults to current logged-in user",
+)
+@click.option(
+    "--sweep_config",
+    "-sc",
+    default=None,
+    help="Path to a sweep configuration yaml",
+)
+@click.option(
+    "--launch_config",
+    "-lc",
+    default=None,
+    help="Path to a launch configuration json/yaml or raw json",
+)
+@click.option(
+    "--resume_id",
+    "-r",
+    default=None,
+    help="Resume a launch sweep by passing an 8-char sweep id. Queue required",
+)
+@click.option(
+    "--num_workers",
+    "-n",
+    default=1,
+    help="Number of concurrent jobs a scheduler can run",
+)
+@display_error
+def launch_sweep(
+    ctx,
+    project,
+    entity,
+    queue,
+    sweep_config,
+    launch_config,
+    resume_id,
+    num_workers,
+):
+    api = _get_cling_api()
+    if api.api_key is None:
+        wandb.termlog("Login to W&B to use the sweep feature")
+        ctx.invoke(login, no_offline=True)
+        api = _get_cling_api(reset=True)
+
+    # Param validation
+    if not (sweep_config or resume_id) or (sweep_config and resume_id):
```

Comment:
is this suppose to be exlusive or i.e. only `sweep_config` or `resume_id`, but not both?
if so maybe modify the message a bit to make it more clear

Reply:
Adding comment, I agree looks busy